app/src/main/java/com/yalta/activities/LoginActivity.kt

```diff
@@ -59,7 +60,11 @@ class LoginActivity : AppCompatActivity() {
     }
 
     private fun successfulLogin() =
-        startActivity(Intent(this@LoginActivity, MainActivity::class.java))
+        if (SessionService.session != null && SessionService.session!!.role is Driver) {
```

Comment:
Yeah. no =) bad practice) 

For everyone who's not driver start admin interface =)) 
better make a `when` statement right from the start even if we only have two roles

Reply:
Fair enough
Updated